.prettierrc.mjs

```diff
@@ -4,5 +4,5 @@ export default {
 	useTabs: true,
 	bracketSpacing: true,
 	htmlWhitespaceSensitivity: 'ignore',
-	plugins: ['prettier-plugin-organize-imports', 'prettier-plugin-tailwindcss'],
+	plugins: ['prettier-plugin-organize-imports', 'prettier-plugin-tailwindcss', 'prettier-plugin-packagejson'],
```

Comment:
```suggestion
	plugins: ['prettier-plugin-organize-imports', 'prettier-plugin-packagejson',  'prettier-plugin-tailwindcss'],
```

Based on last part of README tailwind plugin must come last: https://github.com/tailwindlabs/prettier-plugin-tailwindcss

Reply:
Ah, yes, you are correct! Well remembered! Fixed, thanks!